lib/course_api.rb

```diff
@@ -0,0 +1,193 @@
+require 'terms'
+require 'json'
+require 'nokogiri'
+require 'faraday'
+require 'faraday/retry'
```

Comment:
None of this should be necessary. Rails loads all the dependencies in app/* and Gemfile.

Reply:
Removed the require statements after moving to app/services